core/server/api/authentication.js

```diff
@@ -1,5 +1,5 @@
-var Promise = require('bluebird'),
-    _ = require('lodash'),
+let Promise = require('bluebird'),
```

Comment:
This is a const. If you have variables, which change on runtime, you can use a separate let underneath.